jekyll-assets/_layouts/docs.html

```diff
@@ -11,19 +11,19 @@
         <div id="toc-container">
           <div id="toc-inner">
             <div id="docs-header">
-              <h1 id="docs-header-title">
+              <span id="docs-header-title">
                 <a href="/documentation">
                   Documentation
                 </a>
-              </h1>
+              </span>
               <label class="mobile-menu-toggle" for="mobile-toggle">
                 <div class="mobile-menu-toggle-inner"></div>
               </label>
             </div>
-            <div id="docsearch">
+            <div id="docsearch" role="search">
             </div>
           </div>
-          <div class="toc">
+          <div class="toc" role="navigation">
```

Comment:
If this is truly a navigation element, should we switch the `<div>` for a [`<nav>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/nav)?